web-components/src/components/avatar/Avatar.ts

```diff
@@ -175,46 +198,105 @@ export namespace Avatar {
       this.imageErrored = true;
     }
 
+    handleKeyDown(event: KeyboardEvent) {
+      if (!this.clickable) {
+        return;
+      }
+      const { code } = event;
+      if (code === Key.Enter || code === Key.Space) {
+        this.dispatchEvent(
```

Comment:
we should also check Key.NumpadEnter. It might be worth putting the three checks into a helper method that can be used across components i.e. isActionKey(keyCode)